Cargo.toml

```diff
@@ -18,11 +18,11 @@ base64 = "0.21.0"
 tracing = "0.1"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.79"
-node-bindgen = "5.1.0"
+node-bindgen = { git = "https://github.com/infinyon/node-bindgen", branch = "unsafe-rework" }
```

Comment:
We should only rely on published or master branches (for development work).  Otherwise mark as draft and use only for testing in PR